wai-aria/role/tree-roles.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html>
+<head>
+  <title>Tree related Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<p>Tests <a href="https://w3c.github.io/aria/#tree">tree</a> and related roles.</p>
+
+<ul role="tree" data-testname="role is tree" data-expectedrole="tree" class="ex">
+  <li role="treeitem" data-testname="role is treeitem (in tree)" data-expectedrole="treeitem" class="ex">
+    x
+    <ul role="group" data-testname="role is group (in treeitem)" data-expectedrole="group" class="ex">
+      <li role="treeitem" data-testname="role is treeitem (in group, in treeitem)" data-expectedrole="treeitem" class="ex">x</li>
+      <li role="treeitem">x</li>
+    </ul>
+  </li>
+  <li role="treeitem">x</li>
+</ul>
+
+<table role="treegrid" data-testname="role is treegrid" data-expectedrole="treegrid" class="ex">
```

Comment:
I think this one, or a new example, should include a collapsed row... Check if there is an APG example, but IIRC, that's aria-expanded=false on the row, with either hidden cell contents (e.g. display:none) or without any cell contents...

Reply:
The rest looks good. If you fine the collapsed row examples are difficult for some reason, you could also just include a to-do comment to get that one later. Thanks!

Reply:
Thanks for the suggestion! https://github.com/web-platform-tests/wpt/pull/41519/commits/ba5b36a52ffd4193928f7f65776629221f3063c8 now includes collapsible rows